public/pages/EditRollup/containers/EditRollup.tsx

```diff
@@ -245,6 +254,7 @@ export default class EditRollup extends Component<EditRollupProps, EditRollupSta
       } else {
         //Build JSON string here
         this.updateSchedule();
+        this.updateDelay();
```

Comment:
Seems we just miss this update delay function before? Let's add a unit test for this.

Reply:
I added this updateDelay function just above. It translates from the displayed delay with units to milliseconds for the backend request. It is a good idea to have at least one test for this though so I added a small unit test to make sure that the delay is being displayed correctly